en/option/component/x-axis.md

```diff
@@ -33,13 +33,15 @@ options:
 + `'top'`
 + `'bottom'`
 
-The first x axis in grid defaults to be on the bottom of the grid, and the second x axis is on the other side against the first x axis.
+The first x axis in grid defaults to be on the bottom of the grid, and the second x axis is on the other side against the first x axis.  
+Notice: Changes of `position` cannot be applied unless setting `xAxis.axisLine.onZero` to `false`.
```

Comment:
> Not available when xAxis.axisLine.onZero is `true`

@pissang I think this might not be enough to remind users that `xAxis.axisLine.onZero` is `true` by default  

How about:

> Set `xAxis.axisLine.onZero` to `false` to activate this option.

Reply:
Good point! Your suggestion will be better

Reply:
Thank you for your advice, files updated.